supabase/seed/clientsSeed.mts

```diff
@@ -37,3 +37,5 @@ export const possibleParcelPostCodes = [
     "CB8 9LJ",
     null,
 ];
+
+export const booleansWeightedToTrue = [true, true, true, true, true, true, true, false];
```

Comment:
was thinking of a neat way of writing this but couldn't think of anything nicer than what you have :)